steps/sofy-upload/0.0.2/step.yml

```diff
@@ -0,0 +1,64 @@
+title: Sofy Upload
+summary: |
+  Upload your build at sofy.
+description: |
+  Upload your build at Sofy through Bitrise.
+  You are required to provide the Build Path and Subcription key.
+  You can find your subscription key by logging into Sofy, going to Account Settings --> API Key section.
+  Typically this path is generated by a previous build Step in your
+  Workflow. For example:
+  - The Xcode build for simulator Step generates a .app file at `$BITRISE_APP_DIR_PATH`
+  - The Xcode Archive & Export for iOS Step generates a ipa file at `$BITRISE_IPA_PATH`
+  - The Android Build Step generates apk file at `$BITRISE_APK_PATH`
+website: https://github.com/SOFYAIOFFICIAL/bitrise-step-sofy-upload
+source_code_url: https://github.com/SOFYAIOFFICIAL/bitrise-step-sofy-upload
+support_url: https://github.com/SOFYAIOFFICIAL/bitrise-step-sofy-upload/issues
+published_at: 2021-07-16T17:46:23.898707+05:00
+source:
+  git: https://github.com/SOFYAIOFFICIAL/bitrise-step-sofy-upload.git
+  commit: 3ac70cb97cc13194d8b168bc7cc09357b6bf5c5f
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_path: null
+  opts:
+    description: |
+      A `.app`, `.ipa`, or `.apk` path.
+      Typically this path is generated by a previous build Step in your
+      Workflow. For example:
+      - The Xcode build for simulator Step generates a .app file at `$BITRISE_APP_DIR_PATH`
+      - The Xcode Archive & Export for iOS Step generates a ipa file at `$BITRISE_IPA_PATH`
+      - The Android Build Step generates apk file at `$BITRISE_APK_PATH`
+    is_required: true
+    summary: A `.app`, `.ipa`, or `.apk` path.
+    title: Build Path
+- opts:
+    description: |
+      Each application registered in portal is associated with a unique subcription key.
```

Comment:
This text does not fulfill the purpose of the field. You should rather describe how to generate/obtain that key etc.